integration/test-integration/src/test/resources/apim/config.toml

```diff
@@ -35,6 +35,14 @@ clusterTimeoutInSeconds = 20
 certPath = "/home/wso2/security/keystore/mg.pem"
 keyPath = "/home/wso2/security/keystore/mg.key"
 
+[router.cors]
+enabled = false
+allowOrigins = ["*"]
+allowMethods = ["GET","PUT","POST","DELETE","PATCH","OPTIONS"]
+allowHeaders = ["authorization","Access-Control-Allow-Origin","Content-Type","SOAPAction","apikey", "testKey"]
```

Comment:
shall we add internal-key as well?

Reply:
added via https://github.com/wso2/product-microgateway/pull/1792/commits/3ac56472dcabc5b5e18837bf3442fa5fb021536b